### DISCUSSION TITLE
GitHub Streaks ✨

### DISCUSSION THREAD

**Discussion by DevAesthete** [community/community#65060] [New to GitHub]
Why exactly are Git streaks important on ones profile?

**Comment by helloukeje** [comment]
Streaks are an incredibly powerful way to keep one motivated and steadily progressing toward their goals. But, most importantly, for software developers, your GitHub profile usually is your main portfolio page and having a Git streaks sells you out to be a committed contributor to potential employers.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
